docs/components/tasklist/introduction-to-tasklist.md

```diff
@@ -19,4 +21,11 @@ The GraphQL and REST APIs are currently available, but the GraphQL API will be d
 we recommend using the REST API to ensure long-term compatibility.
 :::
 
+<!--- Are there still plans for deprecation here? --->
```

Comment:
Yes, there is plans for deprecating GraphQL API (it already doesn't have the same functionalities that REST APIs)